go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -156,6 +156,7 @@ func registerTabletEnvFlags(fs *pflag.FlagSet) {
 	flagutil.DualFormatStringVar(fs, &currentConfig.TxThrottlerConfig, "tx_throttler_config", defaultConfig.TxThrottlerConfig, "The configuration of the transaction throttler as a text formatted throttlerdata.Configuration protocol buffer message")
 	flagutil.DualFormatStringListVar(fs, &currentConfig.TxThrottlerHealthCheckCells, "tx_throttler_healthcheck_cells", defaultConfig.TxThrottlerHealthCheckCells, "A comma-separated list of cells. Only tabletservers running in these cells will be monitored for replication lag by the transaction throttler.")
 	fs.IntVar(&currentConfig.TxThrottlerDefaultPriority, "tx-throttler-default-priority", defaultConfig.TxThrottlerDefaultPriority, "Default priority assigned to queries that lack priority information")
+	fs.BoolVar(&currentConfig.TxThrottlerAutoCommit, "tx-throttler-auto-commit", defaultConfig.TxThrottlerAutoCommit, "Whether the transaction throttler acts also on auto-commit statements (i.e. statements issued outside of BEGIN; ... ; COMMIT; blocks)")
```

Comment:
This should not be a choice but should have happened anyways if the throttler is enabled. All the DML queries with or without tx should be throttled if necessary.

The previous change would have been an oversight to be miss throttler check.

Reply:
I am fine to make this always affect all queries; I just thought it would make sense to have it as a flag in order to make the new behavior opt-in, otherwise we are changing it for anyone who might be using it.